ci-operator/step-registry/openshift/upgrade/azure/openshift-upgrade-azure-workflow.yaml

```diff
@@ -3,12 +3,13 @@ workflow:
   steps:
     allow_best_effort_post_steps: true
     pre:
+    - ref: ipi-install-splay
```

Comment:
this happens *before* the install is started?  if so, why is loki in the same spot?

Reply:
Loki install step is dropping manifests into a dir for the installer, which has not yet run. Looks like we should be good.